### PR TITLE
Fix ZonedDateTime example code

### PR DESCRIPTION
**Pull request by ugosan**

### CODE REVIEW

File: docs/painless/painless-guide/painless-datetime.asciidoc [hunk 1]
```diff
@@ -661,7 +661,7 @@ preferred as there is no need to parse it for comparison.
 ----
 long now = params['now'];
 ZonedDateTime inputDateTime = doc['input_datetime'];
```
Comment (abdonpijpelink) [inline docs/painless/painless-guide/painless-datetime.asciidoc:663]:
```suggestion
ZonedDateTime inputDateTime = doc['input_datetime'].value;
```

File: docs/painless/painless-guide/painless-datetime.asciidoc [hunk 2]
```diff
@@ -661,7 +661,7 @@ preferred as there is no need to parse it for comparison.
...
-long millisDateTime = zdt.toInstant().toEpochMilli();
+long millisDateTime = inputDateTime.toInstant().toEpochMilli();
 long elapsedTime = now - millisDateTime;
 ----
 +
```
Comment (abdonpijpelink) [inline docs/painless/painless-guide/painless-datetime.asciidoc:667]: LGTM. Thanks @ugosan !

As long as we're fixing this example, shouldn't `ZonedDateTime inputDateTime = doc['input_datetime']` be `ZonedDateTime inputDateTime = doc['input_datetime'].value`?

**Review by abdonpijpelink — APPROVED** [review]

**Comment by abdonpijpelink** [comment]
@elasticmachine update branch

File: docs/painless/painless-guide/painless-datetime.asciidoc (hunk 2 above)
Comment (ugosan) [reply docs/painless/painless-guide/painless-datetime.asciidoc:667]: You are absolutely right, let me actually review that whole page to make the examples consistent

Comment (abdonpijpelink) [reply docs/painless/painless-guide/painless-datetime.asciidoc:667]: That would be great. Do you want to do that as part of this PR? Or shall we merge this fix, and you can then submit another PR if the review leads to additional changes?

**Pull request merged**